modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -4,36 +4,42 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: mule, requirements, jdk, installation, jre
 
-Recommended minimum hardware is:
+If you plan to install Mule and run it on premises, review these minimum hardware and software requirements before you install.
+
+== Minimum Hardware Requirements
 
 * 2GHz CPU or 1 virtual CPU in virtualized environments
-* 1GB of RAM
+* 1GB of RAM. Add RAM to match your latency requirements, or to support large simultaneous messages or a large number of simultaneous messages if your applications process them.
 * 4GB of storage
 
-Adjust RAM to match your latency requirements and the size and number of simultaneous messages that applications process.
 
-Software requirements are:
+== Supported Software and Systems
+
+Verify that you use a supported version of Java and a supported operating system before you install Mule.
+
+Mule Runtime runs as a standalone server; therefore an application server is not required. However, if you use an application server or need to access a data store, verify that you use a supported application server or database.
+
+[%header,cols="20a,75a"]
+|===
+|Software |Version
+|JDK | JDK 1.8.0 (1.8.0_202 recommended), IBM JDK 1.8, OpenJDK 8 (latest update recommended)
+|OS |MacOS 10.11.4, HP-UX 11i V3, AIX 7.2, Windows 2012 R2 Server, Windows 8.1, Solaris 11.3, RHEL 7, Ubuntu Server 15.04 and 16.04
```

Comment:
@anosenzo @aiannucci , can you guys please validate that these OS and DB versions are current?

Reply:
@anosenzo @aiannucci Could you reconfirm the above details?

Reply:
These are only the versions we have tested the mentioned release against. We cannot say this is required software, because you may also run on a different version of Windows or Linux distributions, that is why we only need to list the software which we tested against, instead of mentioning fixed versions that are supported.

Reply:
Updates:
-Kept _Supported Software_ name since Support team confirm that they support if it's tested.
-Added new Note under Supported Software explaining that the software in the table was used in the testing.
-Left the JDK info under _Required Software_

We also thought of changing the name _Supported Software_ to _Tested Software_
Let us know if you prefer that name.

Reply:
Updated.
Removed Browser information as per Matias Baldini request.